packages/js/src/ui/components/Notification/NotificationList.tsx

```diff
@@ -17,7 +17,7 @@ const EmptyNotificationList = () => {
     <div
       class={style(
         'notificationListEmptyNoticeContainer',
-        'nt-absolute nt-inset-0 nt-flex nt-flex-col nt-items-center nt-m-auto nt-h-fit nt-w-full nt-text-foreground-alpha-100'
+        'nt-flex nt-flex-col nt-items-center nt-m-auto nt-h-fit nt-w-full nt-text-foreground-alpha-100 p-6 text-center'
```

Comment:
I tried adding the `nt-` prefix but I couldn't seem to get the effects to apply. Is there an extra css postbuild step that is required to make the change propagate here?
```suggestion
        'nt-flex nt-flex-col nt-items-center nt-m-auto nt-h-fit nt-w-full nt-text-foreground-alpha-100 nt-p-6 nt-text-center'
```